trainers/image_trainer.py

```diff
@@ -19,6 +19,7 @@ def __init__(self, model, batch_size, save_location,
                 results from training the model.
         """
         super().__init__(model, batch_size, save_location, show_progress_bar)
+        self.noise_size = kwargs.get('conditional_emb_size')
```

Comment:
this is wrong, noise_size and conditional_emb_size are different parameters. One is 100 and the other is 128

Reply:
Very true - thanks